subsys/bluetooth/controller/ll_sw/ti/ti.txt

```diff
@@ -0,0 +1,861 @@
+*** Booting Zephyr OS build zephyr-v2.1.0-2360-ga1f37a008e40  ***
```

Comment:
Looks like a stray file that has been committed by mistake.
If this is added on purpose, please give reason.

Reply:
This is still very much WIP. It's here for the remote TI team on the other side of the planet to have a reference :-) I will definitely remove once the PR is in ready state.